.github/workflows/build-python-package.yml

```
@@ -0,0 +1,53 @@
+name: Build Python Package
+
+on:
+  workflow_call:
+    inputs:
+      package:
+        required: true
+        type: string
+
+jobs:
+    build-python-package:
+        runs-on: ubuntu-22.04
+        steps:
+          - name: Checkout
+            uses: actions/checkout@v3
+
+          - name: Setup Node
+            uses: actions/setup-node@v3
+            with:
+              node-version: '18.x'
+              registry-url: 'https://registry.npmjs.org'
```

We should be using the `.nvmrc` file, which specifies node v20. We should update this in the other actions as well.
```suggestion
          - name: Setup Node
            uses: actions/setup-node@v4
            with:
              node-version-file: '.nvmrc'
              cache: 'npm'
```